src/handbook/index.md

```diff
@@ -2,39 +2,50 @@
 navTitle: Handbook
 ---
 
-## Introduction
+# FlowForge Handbook
 
-This handbook contains all the information about how FlowForge Inc. is run.
-
-It's a living set of documents - they will evolve with time and expand as we learn
+This handbook contains all the information about how FlowForge is run. It's a
+living set of documents - they will evolve with time and expand as we learn
 and discover new things.
 
-The handbook is here for the whole company to help maintain. Pull-requests are welcome
-and strongly encouraged.
+Whilst instinctively we want to be open in all we do, there will inevitably by 
+content that is [not appropriate to make public][data-class]. That content is not
+shared in this handbook.
+
+[data-class]: /handbook/company/security/data-management/#data-classification
 
-### Company
+The handbook is here for the whole company to help maintain. Pull-requests are welcome and [strongly encouraged](#contributing).
 
- - [Values](./company/values.md)
- - [Vision](./company/vision.md)
- - [Operations](./operations)
- - [Product](./product)
+## How FlowForge is run
```

Comment:
Is this the title for the sections below? The ones that start with ##

Reply:
Yes, but open to suggestions though!